main/src/cgeo/geocaching/filters/gui/FilterViewHolderCreator.java

```diff
@@ -77,6 +77,9 @@ private FilterViewHolderCreator() {
             case TERRAIN:
```

Comment:
I think that "TERRAIN" and "DIFFICULTY" should be removed from the type list.

Reply:
see comment above